pyls/workspace.py

```diff
@@ -74,12 +74,12 @@ class Workspace(object):
     M_SHOW_MESSAGE = 'window/showMessage'
     PRELOADED_MODULES = get_preferred_submodules()
 
-    def __init__(self, root_uri, lang_server=None):
+    def __init__(self, root_uri, rpc_manager=None):
```

Comment:
Don't like that everything is a manager.

Reply:
we can talk about this more offline